integration-tests/src/test/java/org/zowe/apiml/integration/zos/ServicesInfoTest.java

```diff
@@ -62,6 +58,11 @@
 
     private static String token;
 
+    @BeforeAll
```

Comment:
Hypothesis around the certificate chain related questions above:
At the moment if the user uses a valid JWT token and reaches the service endpoint via the gateway, it will let him through without having the SAF authority nor the valid certificate.

Reply:
The hypothesis is incorrect.
The gateway's certificate will not be used on the call from gateway. `HttpClientChooser` selects a client without certificate and a JWT will be verified and authorized against SAF.

Reply:
It is a good question to discuss, whether it makes sense to route gateway's endpoints at all, as it serves no purpose, but that was not objective of this PR.

Reply:
This actually also answered the previous questions.